server/services/__tests__/offenderService.spec.js

```diff
@@ -116,6 +116,28 @@ describe('Offender Service', () => {
     });
   });
 
+  describe('getVisitRemaining', () => {
+    it('returns the total visitsRemaining', async () => {
+      const repository = {
+        getVisitBalances: jest
+          .fn()
+          .mockReturnValue({ remainingPvo: 40, remainingVo: 2 }),
+      };
+
+      const service = createPrisonApiOffenderService(repository, {
+        NextVisit: mockAdapter,
+      });
+
+      const data = await service.getVisitsRemaining(TEST_USER);
+
+      expect(repository.getVisitBalances).toHaveBeenCalledWith(
+        TEST_PRISONER_ID,
+      );
+      expect(mockAdapter.from).toHaveBeenCalledWith(RAW_RESPONSE);
```

Comment:
how does this work? You're not using NextVisit are you? 

wonder if there's some kind of test set up issues